### PR TITLE
Allow to set custom repository view class.

### PR DESCRIPTION
**Pull request by lpodkalicki**
This change is [Reviewable](https://reviewable.io/reviews/socialwifi/flask-jsonapi/8)

### CODE REVIEW

**Comment by jgoclawski** [comment]
This conflicts with #9. Let's merge the other one first and then this one.

File: flask_jsonapi/resource_repositories.py [hunk 1]
```diff
@@ -22,34 +22,69 @@ def update(self, id, **data):
         raise exceptions.NotImplementedMethod('Updating is not implemented')
 
 
+class ResourceRepositoryViewMixin:
```
Comment (mimo00) [inline flask_jsonapi/resource_repositories.py]: Move classes as commit "Refactor".

**Review by mimo00 — COMMENTED** [review]
Useful ! 
But, divide it for 2 commits :)

File: flask_jsonapi/resource_repositories.py (hunk 1 above)
Comment (lpodkalicki) [reply flask_jsonapi/resource_repositories.py]: It's a part of this commit as it require class reordering to make it work, I think.

Comment (mimo00) [reply flask_jsonapi/resource_repositories.py]: Sorry, order is important.
+2 :+1:

**Review by mimo00 — APPROVED** [review]

**Pull request merged**